manim/mobject/vector_field.py

```diff
@@ -168,6 +170,25 @@ def construct(self):
         """
         return lambda p: func(p * scalar)
 
+    def fit_to_coordinate_system(self, coordinate_system: CoordinateSystem):
+        """Scale the vector field to fit a coordinate system.
+
+        This method is useful when the vector field is defined in a coordinate system
+        different from the one used to display the vector field.
+
+        This method can only be used once because it transforms the origin of each vector.
+
+        Parameters
+        ----------
+        coordinate_system
+            The coordinate system to fit the vector field to.
+
+        dim
+            The dimension of the vector field. Should be 2 or 3.
```

Comment:
This parameter is documented but not used?

Reply:
Whoops